### DISCUSSION TITLE
Atari vs Amiga

### DISCUSSION THREAD

**Discussion by kovacm** [ArcadeData/arcadedb#269] [General]
Hi,
this is my first post, and it is pretty offtopic (or on edge), but I must ask: 

- who is big fan of Amiga and Atari at ArcadeDB? :)

From ArcadeDB documentation on https://docs.arcadedb.com/#Settings in section "server.defaultDatabases" we have example "'Universe[elon:musk:admin];Amiga[Jay:Miner,Jack:Tramiel]{import:/tmp/movies.tgz}'"

- Jay Miner ("father" of Amiga) and Jack Tramiel (Commodore 64 and Atari ST)...

Sidenote: I start with OrientDB few months ago to build comprehensive timeline database of Jack Tramiel life and his involvement with Commodore, Amiga and Atari Inc/Corp...

**Comment by lvca** [comment]
Hi @kovacm, I am a huge fan of Amiga and Commodore in general!

I started with C16, then C64, Amiga 500 and Amiga 1200.

I never owned an Atari ST, but a friend of mine did and it was always a fight about which one was better :-)

Amiga deserved a much better future than what it had... Anyway, I'd love it if you could use ArcadeDB instead of OrientDB. I am the author of both OrientDB and ArcadeDB. OrientDB is the past, ArcadeDB is the future ;-)

**Reply by kovacm** [reply]
Hi Luca, I read a lot of ArcadeDB in past few days (everything I could find since I am passionate about history, and future ;) ) and highlight fact is that author, you, was an Amiga and Commodore user! :D Many people start their computer careers on Commodore computers and many of them today move the boundaries of computer technologies (like you with OrientDB and now ArcadeDB). Really impressive! 

My computer "path" was: ZX80, Spectrum, Atari ST... beside them, today I own Amiga 500 and 1200 (Amigas was/are great! Especial OS; Amiga OS had interactive icons (e.g. Calendar app shows actual date) long before Mac OS X)... btw I make an Atari ST software archive at http://milan.kovac.cc/atari/software/ 

I will certainly use ArcadeDB for my home projects, but I think it have a potential to replace Neo4J, Google FireStore and Pub/Sub which we use on large, professional project.

Keep up great work!

btw
what is the best place to report e.g. small typos in documentation? Like this "In ArcadeDB it’s mandatory to specify a type when you want **_tot_** create a document, a vertex or an edge."?

**Comment by lvca** [comment]
@kovacm I checked your website and under the serial numbers, I found many models of Atari I didn't even know existed!

You can contribute to the documentation project by forking and sending pull requests, like this one I just merged:

https://github.com/ArcadeData/arcadedb-docs/pull/24

Any contribution is largely appreciated!

**Reply by JohnTMurphy-NIU** [reply]
Love this. My brother insists that the Commodore VIC-20 we had was his, but I don't remember it that way. A couple of weeks ago my dad found a VIC-20 cassette with games on it in an old cabinet. Alas, that computer is long gone.

**Reply by lvca** [reply]
I remember playing with the C-64 Azimuth, a screwdriver, and my cassette reader... Frustrating at the time, now kind of miss the wonder of a new videogame :-) Technically I started with a C=16 from my best friend. That was when I fall in love with programming by using C=16 Basic!
